src/none.js

```diff
@@ -0,0 +1,28 @@
+var _curry2 = require('./internal/_curry2');
+
+
+/**
+ * Returns `true` if no elements of the list match the predicate,
+ * `false` otherwise.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig (a -> Boolean) -> [a] -> Boolean
+ * @param {Function} fn The predicate function.
+ * @param {Array} list The array to consider.
+ * @return {Boolean} `true` if the predicate is not satisfied by every element, `false` otherwise.
+ * @example
+ *
+ *      R.none(isNaN)([1, 2, 3, NaN]); //=> false
+ *      R.none(isNaN)([1, 2, 3]); //=> true
+ */
+module.exports = _curry2(function(fn, list) {
+    var idx = -1;
+    while (++idx < list.length) {
```

Comment:
Accessing the `length` property of an array is O(1), but we're doing it N times. If `list` contains 100 items, caching the value of the `length` property will save 99 property accesses. It's a minor optimization really, but we do it elsewhere and it's nice to be consistent.